packages/flutter/lib/src/cupertino/sliding_segmented_control.dart

```diff
@@ -162,26 +177,21 @@ class _SegmentState<T> extends State<_Segment<T>> with TickerProviderStateMixin<
             child: AnimatedDefaultTextStyle(
               style: DefaultTextStyle.of(context)
                 .style
-                .merge(TextStyle(fontWeight: widget.highlighted ? FontWeight.w500 : FontWeight.normal)),
+                .merge(TextStyle(
+                  fontWeight: widget.highlighted ? FontWeight.w600 : FontWeight.w500,
+                  fontSize: 13,
```

Comment:
Use constants for hardcoded values and document the [provenance](https://github.com/flutter/flutter/issues/21302)?